nighthawk/bazel/nighthawk.bzl

```diff
@@ -0,0 +1,6 @@
+def nighthawk_cc_platform_dep(name):
+    return select({
+        "@envoy//bazel:windows_x86_64": [name + "_win32"],
```

Comment:
We support Windows?! :)

Reply:
Resolving the remark about Windows support for now, but please re-open if this should be addressed